libs/navigation/website/src/lib/navbar/nav-paper.tsx

```
@@ -0,0 +1,246 @@
+  const {hasUser, logout} = useUser()
+  const theme = useTheme()
+
+  return (
+    <NavPaperWrapper>
+      {children && (
+        <NavPaperChildrenWrapper>
+          {iconItems?.links.map(link => {
+            const url = navigationLinkToUrl(link)
+
+            return (
+              <Link href={url} color="inherit">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)